src/com/unbounce/encors/aleph.clj

```diff
@@ -0,0 +1,28 @@
+(ns com.unbounce.encors.aleph
+  "Optional namespace with aleph support. This namespace requires you to have
+  aleph/manifold on the classpath.
+  "
+  (:require [com.unbounce.encors.core :as core]))
+
+;; -- Aleph support  -----------------------------------------------------------
+;; Taken from aleph.http.middleware to optionally support aleph
+(def manifold-enabled?
+  (try
+    (require 'manifold.deferred)
+    true
+    (catch Throwable _ false)))
+
+(defn wrap-cors
+  "Aleph-compatible CORS middleware.
+
+  You should only use this if you have alelph/manifold available on the
+  classpath."
+  {:pre [manifold-enabled?]}
+  [handler cors-policy]
+  (let [deferred? (ns-resolve 'manifold.deferred 'deferred?)
+        chain'    (ns-resolve 'manifold.deferred 'chain')]
+    (core/build-cors-wrapper handler cors-policy
+                             (fn -apply-deferred-headers [response headers]
+                               (if (deferred? response)
+                                 (chain' response #(core/apply-ring-headers % headers))
+                                 (core/apply-ring-headers response headers))))))
```

Comment:
So now that it's in a different ns, you don't need `manifold-enabled` and you can use proper `:require` for `manifold`.

People not having `manifold` on the classpath will not require `com.unbounce.encors.aleph` and thus won't have issues.